.gitignore

```diff
@@ -1,4 +1,5 @@
 *.retry
 community
 gce
+openprovider
```

Comment:
How to ignore folder with inventory file? Do you mean we need another name? What is bad with this name of the folder?